docs/hub/cli/tt_bucket_pull.md

```diff
@@ -1,39 +1,85 @@
 # bucket pull
 
-Pull files and directories from a bucket path. Existing paths will be overwritten. Non-existing paths will be created.
+Pull files and directories from a **bucket path** `[path]` to a **local destination** `[destination]`. Existing paths will be overwritten. Non-existing paths will be created.
 
-Bucket structure is mirrored locally. For example, given the bucket:
-    foo/one.txt
-    foo/bar/two.txt
-    foo/bar/baz/three.txt
+```
+tt bucket pull [path] [destination] [flags]
+```
 
-These 'pull' commands result in the following local structures.
+Bucket structure is mirrored locally. For example, given the **bucket**:
+```
+foo
+│   one.txt
+│
+└───bar
+    │   two.txt
+    │
+    └───baz
+        │   three.txt
+```
 
-'tt bucket pull foo mydir':
```

Comment:
this multiline tabbed format was not showing up in the browser and was really hard to read.